settings.py

```diff
@@ -90,9 +103,9 @@
 if os.path.isfile(ENV_ROOT / "env.json"):
     with open(ENV_ROOT / "env.json") as env_file:
         ENV_TOKENS = json.load(env_file)
-    TMP_GEN_DIR = ENV_TOKENS.get('TMP_GEN_DIR', '/tmp/certificates/')
-    QUEUE_NAME = ENV_TOKENS.get('QUEUE_NAME', 'test-pull')
-    QUEUE_URL = ENV_TOKENS.get('QUEUE_URL', 'https://stage-xqueue.edx.org')
+    TMP_GEN_DIR = ENV_TOKENS.get('TMP_GEN_DIR', TMP_GEN_DIR)
+    QUEUE_NAME = ENV_TOKENS.get('QUEUE_NAME', QUEUE_NAME)
```

Comment:
Wait, does this mean previously `QUEUE_NAME` hadn't been defined?

It looks like if `env.json` existed _but_ it did not contain a value for `QUEUE_NAME`, an exception would be thrown.